README.md

```diff
@@ -279,23 +279,45 @@ Lean code to HTML. This HTML includes hovers, clickable "go to
 definition" links, a search function, and rendered intermediate proof
 states.
 
-There are two steps to using it, after adding a package dependency on
-Verso in your Lake configuration:
+To use this in your project:
 
-1. Generate literate program data from your Lean libraries or modules
+1. Add Verso as a package dependency.
+
+   If you're using a `lakefile.toml` configuration, that looks like this:
+
+   ```toml
+   [[require]]
+   name = "verso"
+   git = "https://github.com/leanprover/verso.git"
+   rev = "latest"
```

Comment:
We should encourage users to pick a tag that corresponds to their Lean version here, esp. because this is oriented towards users for whom Verso/docs are not the main use case.

Reply:
Is there a way to signal this that *doesn't* feel like adding an extra concept that the user has to understand or an effective two additional steps that the user has to perform (step one: "open your lean-toolchain file and note the 'tag', the part after the colon. If your lean-toolchain file contains `leanprover/lean4:v4.25.0-rc2`, then the tag is `v4.25.0-rc2`, step two: "replace the word `"REVISION"` in the sample above with your lean tag, so `"v4.25.0-rc2"` in the example we gave".)

Reply:
(and the thing after the : isn't really the tag _per se_, it's the Lean release. We just use tags that correspond to Lean versions)